openscm/definitions/emissions_units.txt

```diff
@@ -0,0 +1,58 @@
+yr = 1 * year
```

Comment:
I suppose we should also add "a" (annum) as that seems to be unknown to pint...
```suggestion
a = annum = yr = 1 * year
```

Reply:
I always thought "a" is much more common than "yr"...
Also, we should add "h" as for "hour" (see https://github.com/hgrecco/pint/issues/719)

Reply:
can you suggest that?

Reply:
done ;)